### ISSUE TITLE
No uninstall instructions for mamba

### ISSUE THREAD

**Issue by 392781** [conda-forge/miniforge#407]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Looking through the mamba docs, there are only installation instructions and no uninstall instructions.

There is no indication if something like [anaconda-clean](https://docs.anaconda.com/anaconda/install/uninstall/) is applicable or any manual way of uninstalling mamba.

### Installed packages

```shell
N/A
```

### Environment info

```shell
N/A
```

**Comment by hmaarrfk** [comment]
what exactly is missing from those instructions? they provide anaconda-clean, and more extensive manual instructions. Are those missing something?

We also have things documented in: https://github.com/conda-forge/miniforge#uninstallation

**Comment by 392781** [comment]
Yes, but in Mamba docs there's no mention of uninstallation.  It's not exactly clear whether anaconda-clean works with Mamba or not.

Regardless, you shouldn't need to go to the Anaconda documentation to find out how to uninstall Mamba.

**Comment by hmaarrfk** [comment]
Are you trying to uninstall mamba now? 

can you try to follow https://github.com/conda-forge/miniforge#uninstallation

and explain any challenges you are having?

**Comment by 392781** [comment]
I'm not, I'm pointing out an issue with the documentation.

**Comment by hmaarrfk** [comment]
Understood.

If you can think of how to improve the wording of what we have, please feel free to make a PR so we can better discuss.

Help is appreciated.

**Comment by millsks** [comment]
There was one extra steps that I propose to be thorough in #465.

**Comment by jjerphan** [comment]
Can this issue be closed in preference of https://github.com/mamba-org/mamba/issues/2237?
